package.json

```diff
@@ -619,6 +656,14 @@
           "when": "view == iotHubDevices && viewItem == interfaces-label",
           "group": "inline"
         },
+        {
+          "command": "azure-iot-hub.setIoTHub",
```

Comment:
add never for this command in command palette??

Reply:
Added